js/site.js

```diff
@@ -17,14 +19,16 @@ fetch(api)
   .then((data) => {
     console.log(data);
     const {
+      precip_mm,
       temp_f,
       feelslike_f,
       humidity,
       vis_miles,
       wind_mph,
       temp_c,
     } = data.current;
-
+    precip = data.current.precip_mm;
```

Comment:
You can rename a variable when you pull it out of the object above, for example:

```
const { precip_mm: precip, } = data.current;
```